tests/integ/scala/test_dataframe_suite.py

```diff
@@ -181,6 +182,7 @@ def test_view_should_be_updated(session, local_testing_mode):
             Utils.drop_view(session, view_name)
 
 
+@multithreaded_run(num_threads=5)
```

Comment:
what's the criteria of selecting tests to run in multithread mode?

I'm wondering if we can have all tests run in multithreaded_run? it might be an overkill to run the whole test set in each PR but we can also do the whole tests multithread mode in daily running?

Reply:
These tests are selected at random to get diverse coverage of different APIs. No strict rule here. Since each test is run x5, I want to not run this for every test. I think it will be useful for any new apis that are added to simple put this tag and run test their code to get some level of confidence.